USF-2530: Update Wishlist documentation after latest updates (#402)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/wishlist/containers/wishlist.mdx

import OptionsTable from '@components/OptionsTable.astro';

This container manages and displays the list of products on a user's wishlist.

> Configuration options such as `moveProdToCart`, `getProductData` and `getRefinedProduct`,  and routing via `routeEmptyWishlistCTA`, `routeProdDetailPage` and `routeToWishlist` callbacks drive the container's behavior.

Reviewer: ```suggestion
A container's behavior is controlled by configuration options like `moveProdToCart`, `getProductData`, and `refineProduct`, and routing via `routeEmptyWishlistCTA`, `routeProdDetailPage`, and `routeToWishlist`.
```
